.jsonschema.json

```diff
@@ -29,6 +24,35 @@
           "description": "The brand guidelines",
           "$ref": "#/definitions/url"
         },
+        "aliases": {
+          "description": "The aliases for the brand",
+          "type": "object",
+          "properties": {
+            "aka": {
+              "description": "The brand is also known as (e.g. full length name or abbreviation)",
+              "type": "array",
+              "items": { "type": "string" }
+            },
+            "dup": {
+              "description": "Different brands that use the exact same icon",
+              "type": "array",
+              "items": { "$ref": "#/definitions/duplicate" }
+            },
+            "loc": {
+              "description": "Localized names of the brand",
+              "$ref": "#/definitions/locale"
+            },
+            "old": {
```

Comment:
I feel like the renaming strategy should be documented, in contribution guidelines or in some internal document, right?

Reply:
By and large, it's we maintainers that submit the PRs for breaking changes like a brand changing its name so, in the interests of minimising the information in the guidelines, I'd be in favour of omitting it. If an outside contributor does submit a PR to rename an icon, we can ask them to add the alias in the PR.

Reply:
In that case, we should document this internally? I'm a bit loss with all the changes that are taking place lately.

Reply:
Definitely agree it should be documented somewhere (*) but to be frank we haven't really come to a conclusion yet as to what the renaming strategy is complete - specifically how long old names will stay around. So I think it's safe to omit in this PR - as indeed it is mostly just maintainers doing renames - and document it when we agree on a full strategy.

(*): my vote would be to keep it public, just because most everything can and should be public, but I agree that we can choose not to put it here as it is not relevant to most contributors.